squad/frontend/templates/squad/_builds_table.jinja2

```diff
@@ -1,16 +1,24 @@
-<div class='highlight-row'>
+<div class='highlight-row' ng-app='Build' ng-controller='BuildComparisonController'>
+<input type="hidden" ng-init="compare_button_label='{{_('Compare builds')}}'" />
+<input type="hidden" ng-init="invalid_number_of_builds_msg='{{_('Two builds need to be selected!')}}'" />
+<input type="hidden" ng-init="invalid_selected_builds_msg='{{_('Selected builds must be different!')}}'" />
+<input type="button" ng-value="compare_button_label" class="btn btn-default" ng-click="compareBuilds('{{statuses.0.build.project.full_name}}')" />
 {% for status in statuses %}
 {% with build=status.build %}
 <a href="{{build_url(build)}}">
 <div class="row row-bordered build">
+    <div class="col-md-1 col-sm-1">
```

Comment:
this takes too much horizontal space. Add this to the same column as the unfinished/regressions/fixes column, but inside a div that has a fixed width so that the entire row doesn't move around when the second radio button is not visible.

Reply:
had to remove some space from the last column (with the time), and added the radio buttons inside the unfinished/regressions/fixes columns, each surrounded by a `div.row`, this way it guarantees that the rest of columns wouldn't move. Couldn't find a better way for it to take less horizontal space though